lib/graphics.js

```diff
@@ -392,19 +392,24 @@ function graphics(callback) {
     if (_windows) {
       try {
         const basePath = util.WINDIR + '\\System32\\DriverStore\\FileRepository';
-        // find all directories that have an nvidia-smi.exe file
-        const candidateDirs = fs.readdirSync(basePath).filter(dir => {
-          return fs.readdirSync([basePath, dir].join('/')).includes('nvidia-smi.exe');
-        });
-        // use the directory with the most recently created nvidia-smi.exe file
-        const targetDir = candidateDirs.reduce((prevDir, currentDir) => {
-          const previousNvidiaSmi = fs.statSync([basePath, prevDir, 'nvidia-smi.exe'].join('/'));
-          const currentNvidiaSmi = fs.statSync([basePath, currentDir, 'nvidia-smi.exe'].join('/'));
-          return (previousNvidiaSmi.ctimeMs > currentNvidiaSmi.ctimeMs) ? prevDir : currentDir;
-        });
+        let newestNvidiaSmi = null;
+        for (const dir of fs.readdirSync(basePath)) {
+          // nvidia-smi will only be located in directories starting with `nvdm`.
```

Comment:
this may or may not be true. we may want to broaden this to just look inside all directories that start with `nv` (case insensitive.)